database/forms_old.py

```diff
@@ -60,7 +60,7 @@ def clean_species(self):
         try:
             molecule = Molecule()
             molecule.fromAdjacencyList(str(self.cleaned_data['species']))
```

Comment:
It looks like 2to3 missed this line. Not part of the PR but we should probably doublecheck.

Reply:
oh, I didn't run it through the RMG 2to3 thing (I should have. Oops). Just the in-buit python one. My VSCode IDE was complaining about syntax errors in a bunch of unused files and the refactoring tools weren't working, so I did the bare minimal to make them go away. 

I guess from its name that this forms_old isn't used? 
if trivial to run through the RMG 2to3 then by all means do, but wouldn't spend long if this is indeed dead code.

Reply:
Yea, I wouldn't say that it's _that_ important for this file. I'm going to run 2to3 on files that import rmgpy and then push. If we run into errors in the future, we can fix them as they pop up. But other than that this should be good to merge in.